testdata/project-v4-with-deploy-image/dist/install.yaml

```diff
@@ -646,6 +646,7 @@ spec:
     spec:
       containers:
       - args:
+        - --metrics-bind-address=:8443
```

Comment:
See that now we have the bind again to allow the bind when the metrics is patched
However, by default we are enabling the patch but it allows they disable the feature and have no metrics server if they comment the default/kustomize